Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM public.ecr.aws/lambda/python:3.12
+
+RUN dnf install -y atk cups-libs gtk3 libXcomposite alsa-lib \
+    libXcursor libXdamage libXext libXi libXrandr libXScrnSaver \
+    libXtst pango at-spi2-atk libXt xorg-x11-server-Xvfb \
+    xorg-x11-xauth dbus-glib dbus-glib-devel nss mesa-libgbm jq unzip
+
+COPY ./chrome_installer.sh ./chrome_installer.sh
+RUN ./chrome_installer.sh
+RUN rm ./chrome_installer.sh
+
+COPY requirements.txt ${LAMBDA_TASK_ROOT}
```

Comment:
Is the `LAMBDA_TASK_ROOT` automatically set? Or does the user have to do that?

Reply:
Automatically set